devtools/src/main/scala/kalix/devtools/impl/DockerComposeUtils.scala

```diff
@@ -78,6 +78,11 @@ case class DockerComposeUtils(file: String, envVar: Map[String, String]) {
       Process(s"docker-compose -f $file stop", None).run()
     }
 
+  def stopAndWait(): Int =
+    if (started) {
+      Process(s"docker-compose -f $file stop", None).!
+    } else 0
+
```

Comment:
While preparing the scala protobuf tests, I observed that after running the tests, some containers may be still running.

We can use the `stopAndWait` method wait for the stop to finish and compose it in a future in the after all. 

I will rebase the Scala branch on this one and use it there as well.